webapp/components/create_post.jsx

```diff
@@ -503,7 +503,7 @@ class CreatePost extends React.Component {
         }
 
         let centerClass = '';
-        if (this.state.centerTextbox) {
+        if (!this.state.centerTextbox) {
```

Comment:
Why's the logic being reversed here? Doesn't seem to make sense to add a `center` class when the state says not to center